front_end/core/sdk/ReactNativeApplicationModel.ts

```diff
@@ -21,14 +22,26 @@ export class ReactNativeApplicationModel extends SDKModel<EventTypes> implements
     this.#enabled = false;
     this.#agent = target.reactNativeApplicationAgent();
     target.registerReactNativeApplicationDispatcher(this);
+
+    // Auto-init. Paired with registering this model immediately in rn_fusebox.ts.
+    this.ensureEnabled();
   }
 
   ensureEnabled(): void {
     if (this.#enabled) {
       return;
     }
 
-    void this.#agent.invoke_enable();
+    void this.#agent.invoke_enable()
+      .then(result => {
+        const maybeError = result.getError();
+        const success = !maybeError;
+        Host.rnPerfMetrics.fuseboxSetClientMetadataFinished(success, maybeError);
+      })
+      .catch(reason => {
+        const success = false;
+        Host.rnPerfMetrics.fuseboxSetClientMetadataFinished(success, reason);
+      });
```

Comment:
I believe you are missing `Host.rnPerfMetrics.fuseboxSetClientMetadataStarted();` somewhere here

Reply:
Yes! Updated.